src/Catrobat/Controller/Web/DefaultController.php

```diff
@@ -160,4 +162,29 @@ public function makeNonRecommendedProgramClickStatisticAction(Request $request):
 
     return new Response('error');
   }
+
+  /**
+   * @Route("/checkFirstOauthLogin", name="oauth_first_login", methods={"GET"})
+   */
+  public function checkOauthFirstLogin(): Response
+  {
+    /** @var User|null $user */
+    $user = $this->getUser();
+    $user_first_login = false;
+    $user_id = null;
+    if (null !== $user)
+    {
+      if (true == $user->isOauthUser() && !$user->isOauthPasswordCreated())
+      {
+        $user_first_login = true;
+        $user_id = $user->getId();
+      }
+    }
+
+    return JsonResponse::create([
+      'statusCode' => Response::HTTP_OK,
```

Comment:
a HTTP response already hast a response code. No need to artificially add this 'statusCode' entry

Reply:
I think it is the second param. of a JsonResponse